### PR TITLE
Updating diagrams and text on Getting Started page

### PR DESCRIPTION
**Pull request by alextha-unity**
Now that we've expanded our offering, we have multiple paths through the product now, which will increase as we go further.  For example, the dataset evaluation instructions can be followed with an official or generated synthdet dataset.  Adding text and diagrams to clarify, which can be extended to content creation and AR app.

### CODE REVIEW

**Comment by alextha-unity** [comment]
Note that I will squash-merge this branch.  It's a bit goofy because I tried to rebase to master after branching from a branch that was many commits ahead, which brought most of my commits with me, but they were easy to clean up.

File: docs/Readme.md [hunk 1]
```diff
@@ -1,12 +1,15 @@
 # SynthDet Documentation
+<img src="images/Synthetic Data pipeline-Perception Workflow.png" align="middle"/>
```
Comment (JonathanHUnity) [inline docs/Readme.md:2]: What is the difference between Dataset Evaluation and Model Evaluation? Afaict, there is only one type of evaluation, which evaluates a trained model on a target dataset. Also, Dataset Inspection/Statistics feeding a dataset into Dataset Evaluation is a bit odd. Consider drawing a box around Instpection/Statistics and Evaluation and label it datasetinsights. The dataset (that is, the directory of files) can be fed into any of those boxes.

File: docs/Readme.md [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
 ## Installation & Setup
 * [Prerequisites](Prerequisites.md)
 
-## Getting Started 
-* [Getting Started with SynthDet](GettingStartedSynthDet.md)
-* [Running SynthDet in Unity Simulation](RunningSynthDetCloud.md)
+## Getting Started with data generation in Unity
+* Data generation example project - [Getting Started with SynthDet](GettingStartedSynthDet.md)
+* Scaling up data generation - [Running SynthDet in Unity Simulation](RunningSynthDetCloud.md)
+* Evaluating a dataset - [Dataset Evaluation with Dataset Insights framework](https://github.com/Unity-Technologies/datasetinsights/blob/master/docs/source/Evaluation_Tutorial.md)
```
Comment (JonathanHUnity) [inline docs/Readme.md:10]: Should this be Dataset Insights or datasetinsights?

File: docs/Readme.md (hunk 1 above)
Comment (JonathanHUnity) [reply docs/Readme.md:2]: I guess it isn't clear if this is a data flow diagram or a user flow diagram.

**Review by JonathanHUnity — APPROVED** [review]
Looks good, with a couple comments.

File: docs/Readme.md (hunk 2 above)
Comment (alextha-unity) [reply docs/Readme.md:10]: Could go either way, but on https://github.com/Unity-Technologies/datasetinsights the title text says "Dataset Insights"

Comment (adason) [reply docs/Readme.md:10]: https://datasetinsights.readthedocs.io/en/latest/Evaluation_Tutorial.html We should use this rendered version of official datasetinsights doc. It also includes documents for the datasetinsighs modules.

**Review by adason — CHANGES_REQUESTED** [review]

File: docs/Readme.md (hunk 2 above)
Comment (alextha-unity) [reply docs/Readme.md:10]: Agreed, that was the next step, I just wanted to make sure we agreed on this sort of diagram first.  I already have the png for the DI page

File: docs/Readme.md (hunk 1 above)
Comment (alextha-unity) [reply docs/Readme.md:2]: Updated diagrams per discussion

**Review by adason — APPROVED** [review]

**Pull request merged**